packages/expo-dev-launcher/plugin/src/withDevLauncher.ts

```diff
@@ -233,7 +234,10 @@ const withDevLauncherPodfile: ConfigPlugin = config => {
 const withErrorHandling: ConfigPlugin = config => {
   const injectErrorHandlers = async (config: ExportedConfigWithProps) => {
     await editIndex(config, index => {
-      if (!index.includes(DEV_LAUNCHER_JS_REGISTER_ERROR_HANDLERS)) {
+      if (
+        !index.includes(DEV_LAUNCHER_JS_REGISTER_ERROR_HANDLERS) &&
+        !index.includes(DEV_LAUNCHER_JS_REGISTER_ERROR_HANDLERS_VIA_LAUNCHER)
+      ) {
         index = DEV_LAUNCHER_JS_REGISTER_ERROR_HANDLERS + '\n\n' + index;
```

Comment:
```suggestion
        index = DEV_LAUNCHER_JS_REGISTER_ERROR_HANDLERS + ';\n\n' + index;
```
To be consistent with the style of index.js in newly prebuilt projects, which does include semicolons.

Reply:
Are you sure that we don't use semicolons in `index.js` anymore?

Reply:
I see them here https://github.com/expo/expo/blob/master/templates/expo-template-bare-minimum/index.js which is also what I see when I run expo init (blank) -> expo prebuild

Reply:
Sorry, I for some reason thought that I should remove the semicolon not add it 🤦‍♂️ You are right. Nice find 🥇